shared/src/TwinJsonConverter.cs

```diff
@@ -39,6 +39,7 @@ public sealed class TwinJsonConverter : JsonConverter
         private const string CloudToDeviceMessageCountTag = "cloudToDeviceMessageCount";
         private const string AuthenticationTypeTag = "authenticationType";
         private const string X509ThumbprintTag = "x509Thumbprint";
+        private const string ModelId = "digital-twin-model-id";
 
```

Comment:
Is this change working now? This is expected to change fyi.

Reply:
Isn't the change specific to what we send on the mqtt connect packet? This shouldn't be affected in that case.
Or are we talking about a different change here?

Reply:
Yes it is working. 1) There is backward compatibility as existing tests are passing 2) This has been manually tested using samples against the hub they provided

Reply:
Great that it is working. Then lets take the followup change in next PR. The rename applies to both device and service SDK's.